catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/OperationImpl.java

```diff
@@ -21,7 +21,7 @@
 
 public abstract class OperationImpl implements Operation {
 
-  protected Map<String, Serializable> properties;
+  protected transient Map<String, Serializable> properties;
```

Comment:
❓ Doesn't hurt anything, but do we know why Coverity needs `transient` here? I thought it was because of the `Serializable` values in the `Map`, but there are other collections that contain `Serializable` objects in the other classes and afaict, they haven't been flagged.
❓ Can we also make the field `private` while we're at it?

Reply:
I tried that on a couple of other fields along with this one and it broke the build locally when all of my changes were combined. I can try your suggested accessibility change in isolation though now that the build is working with my other changes.

Reply:
Just verified that making the field private breaks the build:
ddf/catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/faceting/FacetedQueryRequest.java:[71,9] properties has private access in ddf.catalog.operation.impl.OperationImpl

Reply:
@lessarderic We could make the var private, but create a protected accessor for it. What do you think?

Reply:
@lessarderic should I make the changes that @ahoffer is suggesting above?

Reply:
That'd work for me. Thanks!